### PR TITLE
Created WebGL heatmap doc for R API

### PR DESCRIPTION
**Pull request by priyatharsan**
R tutorial for WebGL Heatmap created with ref to issue #740 
@cldougl @bcdunbar

### CODE REVIEW

File: _posts/r/scientific/2018-08-03-heatmap-webgl.Rmd [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+tmpf <- tempfile()
+download.file(url,tmpf,mode="wb")
+data <- readJPEG(tmpf)
+file.remove(tmpf) # remove the downloaded temp file
+
+zdata = rowSums(data*255, dims = 2)
+
+p <- plot_ly(
+            z = zdata,
+            colorscale = list(c(0,1),c("rgb(155,112,184)","rgb(128,100,64)")),
```
Comment (bcdunbar) [inline _posts/r/scientific/2018-08-03-heatmap-webgl.Rmd]: I'd probably make the colors pop a little more, something like you have but `colorscale = list(c(0,0.5,1),c("blue", "white", "red"))`

Comment (priyatharsan) [reply _posts/r/scientific/2018-08-03-heatmap-webgl.Rmd]: Thanks! @bcdunbar  I have changed them to the suggested colorscale.

**Comment by bcdunbar** [comment]
@priyatharsan Great 👍 lgtm 💃

**Comment by priyatharsan** [comment]
Thanks!

**Pull request merged**